package.json

```diff
@@ -43,6 +43,7 @@
     "gatsby-plugin-sitemap": "^2.4.5",
     "gatsby-source-airtable": "^2.1.1",
     "gatsby-source-filesystem": "^2.3.10",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
Since we're leveraging the API can we remove any of the algolia packages from here? I believe it's just the one on line 29 :)

Reply:
Totally forgot about that, will do that!